.changeset/tricky-hairs-double.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/catalog-backend-module-aws': minor
```

Comment:
Thanks for taking this on! I haven't looked at the rest of modifications yet but this looks like a breaking change so let's make it `major`.

Reply:
Hello @Xantier 
I'm working to manage backward compatiblity right now, better to turn this PR as draft.